### PR TITLE
Fixes cyborg pistol(new sprite)

### PR DESCRIPTION
**Pull request by Gandalf2k15**
Used to flash empty at all times. Replaced the sprite.
image
Fixes https://github.com/HippieStationCode/HippieStation13/issues/2481

### CODE REVIEW

**Comment by Spacedong** [comment]
looks really gay and the ammo counter looks unshaded? why?

**Comment by Gandalf2k15** [comment]
Because you fucking asked for it Also the ammo shader is shaded just not in that pic

**Comment by Kokojos** [comment]
Spacedong is never happy, ever.

**Comment by Gandalf2k15** [comment]
Take it and go

**Comment by Spacedong** [comment]
take it and go you can improve the sprites later

**Comment by Kokojos** [comment]
Headmuns, votes.

+1/3 from me, let's at least try it.

**Comment by ghost** [comment]
So this fixes it by making it a static sprite or what?

**Comment by Gandalf2k15** [comment]
Yes

File: code/modules/projectiles/guns/energy/laser.dm [hunk 1]
```diff
@@ -99,6 +101,7 @@
 	item_state = "laser"
 	w_class = 4
 	force = 10
+	stamina_percentage = 0.4
```
Comment (TDcoolguy300) [inline code/modules/projectiles/guns/energy/laser.dm:104]: Whats up with this change?

File: code/modules/projectiles/guns/energy/laser.dm [hunk 2]
```diff
@@ -30,6 +30,7 @@
 	item_state = "caplaser"
 	desc = "This is an antique laser gun. All craftsmanship is of the highest quality. It is decorated with assistant leather and chrome. The object menaces with spikes of energy. On the item is an image of Space Station 13. The station is exploding."
 	force = 10
+	stamina_percentage = 0.6
```
Comment (TDcoolguy300) [inline code/modules/projectiles/guns/energy/laser.dm:33]: This too

File: code/modules/projectiles/guns/energy/laser.dm (hunk 1 above)
Comment (Gandalf2k15) [inline code/modules/projectiles/guns/energy/laser.dm:104]: I don't actuallly know.

Comment (TDcoolguy300) [inline code/modules/projectiles/guns/energy/laser.dm:104]: Nevermind me, they were merged in another PR

**Comment by TDcoolguy300** [comment]
This is techincally bug fixes, so merging

**Pull request merged**